hack/README.md

```diff
@@ -19,39 +19,32 @@ diff --git a/hack/update-generated-code.sh b/hack/update-generated-code.sh
  #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
  #                  instead of the $GOPATH directly. For normal projects this can be dropped.
  bash ${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
--  github.com/kubernetes-csi/external-snapshotter/pkg/client github.com/kubernetes-csi/external-snapshotter/pkg/apis \
-+  github.com/kubernetes-csi/external-snapshotter/v2/pkg/client github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis \
-   volumesnapshot:v1beta1 \
+   github.com/kubernetes-csi/external-snapshotter/client/v2 github.com/kubernetes-csi/external-snapshotter/client/v2/apis \
+  volumesnapshot:v1beta1 \
```

Comment:
This section talks about you have to manually modify update-generated-code.sh.  I thought we don't have to manually modify the script any more?

Reply:
@xing-yang yes, when i run this script i don't modify it manually, so accordingly i am removing the section from README